common/v2/components/WalletUnlock/Web3ProviderInstall.tsx

```diff
@@ -16,7 +16,9 @@ import './Web3ProviderInstall.scss';
 function InstallTrunk() {
   return (
     <div className="Panel">
-      <div className="Panel-title">{translate('ADD_ACCOUNT_WEB3_INSTALL_TITLE')}</div>
+      <div className="Panel-title">
+        {translate('ADD_ACCOUNT_WEB3_INSTALL_TITLE', { $walletId: 'Trunk' })}
```

Comment:
Shouldn't this be "Trust"?

Reply:
Ah, my bad. But the problem is that there are two choices. I'll ask in Slack.